src/controlnet_aux/lineart/__init__.py

```diff
@@ -122,7 +123,11 @@ def to(self, device):
         self.model_coarse.to(device)
         return self
     
-    def __call__(self, input_image, coarse=False, detect_resolution=512, image_resolution=512, return_pil=True):
+    def __call__(self, input_image, coarse=False, detect_resolution=512, image_resolution=512, return_pil=None, output_type="pil"):
```

Comment:
This would break if the user was calling positionally?  Not following how this helps with backwards compatibility.

Reply:
How are backward compatibilities typically solved? This seems like something that occurs a lot

Reply:
Here, I'd argue that `"output_type"` replaces `return_pil` so in that case the position stays the same (since `output_type` takes the position of `return_pil`. Now to still support `return_pil` we add a ``kwargs`` argument. That's very common practice to deprecate function arguments. 

To be fully backwards compatible we could in addition add the following check:

```py
if type(output_type) is bool:
    warnings.warn("Passing `True` or `False` to `output_type` is deprecated and will raise an error in future versions")
    if output_type:
        output_type="pil"
```

Reply:
Thanks, makes sense!